src/server/auth/server/api_server.go

```diff
@@ -53,9 +53,6 @@ const (
 	// GitHookPort is 655
 	// Prometheus uses 656
 
-	// OidcPort is the port where OIDC ID Providers can send auth assertions
-	OidcPort = 657
```

Comment:
I think we are right to move this out of here, but I think it should just go into the deploy package.  The arbitrary config values should not be spread across the codebase.

Values like ports don't matter in isolation, they could be anything.  They matter, and become non-arbitrary when constrained by other port choices.  That happens in the deployment configuration, so that's where we should pick port numbers.